lib/mix/phx_gen_auth/hashing_library.ex

```diff
@@ -3,38 +3,38 @@ defmodule Mix.Phx.Gen.Auth.HashingLibrary do
 
   defstruct [:module, :mix_dependency, :test_config]
 
-  def build("bcrypt") do
```

Comment:
Can you please revert the changes to this file? We don't need to reorder these clauses, do we? :)